study-builder/checkstyle.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
```

Comment:
Don't know why we decided to duplicate the checkstyle file for study-builder. Oh well. These changes syncs up this file with the one in `pepper-apis` directory.